### PR TITLE
Tlsa/bloodview transmissive

### PR DESCRIPTION
**Pull request by tlsa**
Enable configuration of transmissive mode from Bloodview.

### CODE REVIEW

File: bloodview/src/main-menu.c [hunk 1]
```diff
@@ -437,6 +440,12 @@ static const cyaml_strval_t bv_acq_emission_mode[] = {
 	{ .val = BL_ACQ_FLASH,      .str = "Flash" },
 };
 
+/** CYAML schema: Valid acquisition detection mode mapping. */
+static const cyaml_strval_t bv_acq_detection_mode[] = {
+	{ .val = BL_ACQ_CONTINUOUS, .str = "Reflective" },
```
Comment (kplus) [inline bloodview/src/main-menu.c:445]: .val filed messed up with emission-mode

Comment (tlsa) [reply bloodview/src/main-menu.c:445]: Ooh, good spot!

Comment (tlsa) [reply bloodview/src/main-menu.c]: Fixed.

**Comment by kplus** [comment]
looks good

**Pull request merged**